### PR TITLE
[ja]  /routing full page translation

### PR DESCRIPTION
**Pull request by koyakei**

### CODE REVIEW

File: ja/development/routing.rst [hunk 1]
```diff
@@ -1,27 +1,1146 @@
...
+    // 静的メソッドを使います。
+    Router::connect('/', ['controller' => 'Articles', 'action' => 'index']);
+
+``Router`` はルーティングするための2つのインターフェイスを提供します。
+この静的メソッドはカッコ付きビルダーで複数のルーティングをするための
+より短い表現方法をもつ後方互換性のあるより良い性能のインターフェイスです。
+
+これはサイトのホームページにアクセスした時に ``ArticlesController`` の index メソッドを実行します。
+時々、複数のパラメーターを受け取る。動的ルーティングが必要になると思います。
+それが必要になるケースは、この現在見ている article(記事)のコンテンツへのルーティングです。
```
Comment (antograssiot) [inline ja/development/routing.rst]: missing `::`

File: ja/development/routing.rst [hunk 2]
```diff
@@ -1,27 +1,1146 @@
...
+``:controller`` のようなルーティング要素を含むテンプレートを使用している場合、
+``Router::url()`` にオプションの一部としてそれらを提供したい場合があると思います。
+
+.. note::
+
+    ルーティング名をアプリケーション全てで一意にする必要があります。
+    違うルーティングスコープ出ない限り、同じ ``_name`` を二度使えません。
+
+名前付きルーティングをさくせうする時、ルーティング名にいくつかの命名規則を適用したいでしょう。
+ CakePHP はそれぞれのスコープのプレフィックス名を定義することで、より簡単にルーティング名をきめられます。
```
Comment (antograssiot) [inline ja/development/routing.rst]: `::`

File: ja/development/routing.rst [hunk 3]
```diff
@@ -1,27 +1,1146 @@
...
+        ['_name' => 'login']
+    );
+
+    echo Router::url(['_name' => 'login']);
+    // 出力
+    /login
+
+ルーティングコードをドライに保つために、ルーターは 'scopes' （スコープ）というコンセプトを持っています。
+スコープは一般的なパスセグメントを定義し、オプションとして、デフォルトにルーティングします。
+ すべてのスコープの内側に接続されているルートは、ラップしているスコープのパスとデフォルトを継承します。::
```
Comment (antograssiot) [inline ja/development/routing.rst]: extra space at the beginning of the line. (also in other places in the file)

File: ja/development/routing.rst [hunk 4]
```diff
@@ -1,27 +1,1146 @@
...
+いくつかの特別な意味を持つルーティング要素があります。
+そして、特別な意味を持たせたくないなら、使ってはいけません。
+
+* ``controller`` コントローラー名に使います。
+* ``action`` アクション名に使います。
+* ``plugin`` コントローラーにあるプラグイン名に使います。
+* ``prefix`` :ref:`prefix-routing` のために使います。
+* ``ext`` :ref:`file-extensions` ルーティングのために使います。
+* ``_base`` false にセットすると、ベースURLを除去します。
+      ルートディレクトリに作っているアプリがない場合、 'CakePHP relative' なURLの生成に使えます。
```
Comment (antograssiot) [inline ja/development/routing.rst]: indentation seems off here

File: ja/development/routing.rst (hunk 1 above)
Comment (koyakei) [inline ja/development/routing.rst]: thanks, I ffixed. now my vagrant is not working so fix it by hand.

**Comment by antograssiot** [comment]
other warnings fixed in fcfb754ec9a97df3388e291d314ddb0997e3e4eb
thanks @koyakei

**Pull request merged**